sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
-## 12.0.0b1 (Unreleased)
+## 12.0.0b2 (Unreleased)
```

Comment:
I wouldn't include an entry for the next release, once you afctually release the service, we have an automated PR that updates the versioning